README.md

```diff
@@ -85,6 +88,165 @@ public class TokenGenerationUtil {
     }
 }
 ```
+## Service Account Bearer Token with Context Generation 
+
+The service account generated with `context_id` identifier enabled can be used to generate bearer tokens with `context`.  The token generated from this service account is valid for 60 minutes and can be used to make API calls to vault services as well as management API(s) based on the permissions of the service account.
```

Comment:
This first sentence "The service account generated with `context_id` identifier enabled can be used to generate bearer tokens with `context`." doesn't make sense to me.

What is a context_id identifier and as someone outside of Skyflow, how would I know what that is?

Reply:
To add onto this, what is `context`, and why would I want it?